liberty-archetype-mp/src/main/resources/archetype-resources/pom.xml

```diff
@@ -0,0 +1,115 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>${groupId}</groupId>
+    <artifactId>${artifactId}</artifactId>
+    <packaging>war</packaging>
+    <version>${version}</version>
+
+    <parent>
+        <groupId>net.wasdev.wlp.maven.parent</groupId>
+        <artifactId>liberty-maven-app-parent</artifactId>
+        <version>${wlpPluginVersion}</version>
+    </parent>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+        <maven.compiler.source>1.8</maven.compiler.source>
```

Comment:
I think you'll need to change this to 1.7 for compatibility with Travis

Reply:
This is a bit bizarre as I copied this from the other archetypes.  I'll do this, but it shouldn't be failing so far as I know.  
I noticed that travis and appveyer are still using 17.0.0.3 :(

Reply:
Yeah, I think most of the issues I brought up in this PR also exist for our other archetypes. I never reviewed the code for those. 

Anyways, I believe the reason that it's working in the other archetypes is because the archetype projects are never built in the Travis tests. This was done when you added `clean install` to `goal.txt`. If you look at the commit history you'll notice that Travis was passing before 393012f35b72824daf55f2d0ee90d8864e272bce.

Reply:
Ah, that would explain it.  OK, I'll drop the version to 1.7